src/style/color-config/index.js

```diff
@@ -6,6 +6,7 @@ export default {
   genericGreen: '009900',
   productGreen: '00A376',
   productBlue: '0077C8',
+  lightBlue: '1963F6',
```

Comment:
is it used in the classic theme only?

Reply:
Yep, sorry updated now - put it there before realising it wasn't used elsewhere 👍